pkg/server/router/credential_test.go

```diff
@@ -26,6 +27,9 @@ import (
 
 func TestCredentialRouter(t *testing.T) {
 
+	config.SetAPIBase("http://localhost:1234")
```

Comment:
`apiBase` is a global setting, but there are two tests (maybe more) setting it. Tests in different files run concurrently in golang, so test may fail when they're run because they both modify the same variable.

Having a single static variable that's not injectable, makes it hard to test. Consider passing a reference to the singleton as a parameter to `NewCredentialService`. This way, you can then test using a simple mock.

Reply:
I would rather not introduce dependencies to each service on what's supposed to be a global config
as an alternative I'll refactor to have all tests use the same API base

Reply:
another option is refactoring the Set methods to SetOnce semantics so they cannot be overwritten